overops-reset-cache/values.yaml

```diff
@@ -0,0 +1,33 @@
+# Default values for overops-reset-cache.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+cache:
+  host: ''
+  port: 6379
+
+image:
+  repository: redis
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "6.2-alpine"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
```

Comment:
This more for reference when and if you added these values?

Reply:
So these come from the default template whenever you run 
`helm create blah`
which basically scaffolds a chart for you. I can remove it, its really just an example but won't really apply for this chart.

Reply:
@DFieldFL Fixed